src/main/java/com/linkedin/kmf/common/Utils.java

```diff
@@ -93,7 +93,9 @@ public static int createTopicIfNotExists(String topic, short replicationFactor,
       int partitionCount = Math.max((int) Math.ceil(brokerCount * partitionToBrokerRatio), minPartitionNum);
       try {
         NewTopic newTopic = new NewTopic(topic, partitionCount, replicationFactor);
+        //noinspection rawtypes
         newTopic.configs((Map) topicConfig);
+        @SuppressWarnings("rawtypes")
         List topics = new ArrayList<NewTopic>();
```

Comment:
why not change this declaration to List<NewTopic> ?

Reply:
sgtm